cob_hardware_config/cob3-2/config/joint_limits.yaml

```diff
@@ -0,0 +1,99 @@
+# joint_limits.yaml allows the dynamics properties specified in the URDF to be overwritten or augmented as needed
```

Comment:
yep! maybe the files on cob_moveit_config can be deleted, I am not sure because cob_moveit_config is a package generated automatically by moveit_setup_assistant maybe in case that we need to modify something using the assistant and we want to load our current model  is required that the files are there(?)